charts/bivac/values.yaml

```diff
@@ -0,0 +1,50 @@
+images:
```

Comment:
It looks like common pattern on github.com/helm/charts id to specify image repository and tag as `image.repository` and `image.tag`. I think we should use the same pattern.